diff -ur planner-0.14.6/src.orig/planner-eds-plugin.c planner-0.14.6/src/planner-eds-plugin.c
--- planner-0.14.6/src.orig/planner-eds-plugin.c	2011-12-22 22:53:15.000000000 +0000
+++ planner-0.14.6/src/planner-eds-plugin.c	2013-04-08 22:37:45.073592620 +0100
@@ -152,8 +152,8 @@
 
 static GtkActionEntry action_entries[] = {
 	{ "EDS Import", NULL,
-	  N_("EDS"), NULL,
-	  N_("Import resources from Evolution Data Server"),
+	  "EDS", NULL,
+	  "Import resources from Evolution Data Server",
 	  G_CALLBACK (eds_plugin_import) },
 };
 static guint n_action_entries = G_N_ELEMENTS (action_entries);
diff -ur planner-0.14.6/src.orig/planner-gantt-view.c planner-0.14.6/src/planner-gantt-view.c
--- planner-0.14.6/src.orig/planner-gantt-view.c	2011-09-29 15:15:45.000000000 +0100
+++ planner-0.14.6/src/planner-gantt-view.c	2013-04-09 08:13:28.611700756 +0100
@@ -135,64 +135,64 @@
 static PlannerViewClass *parent_class = NULL;
 
 static const GtkActionEntry entries[] = {
-	{ "InsertTask",      "planner-stock-insert-task",    N_("_Insert Task"),
-	  "<control>i",        N_("Insert a new task"),
+	{ "InsertTask",      "planner-stock-insert-task",    ("_Insert Task"),
+	  "<control>i",        ("Insert a new task"),
 	  G_CALLBACK (gantt_view_insert_task_cb) },
-	{ "InsertTasks",     "planner-stock-insert-task",    N_("In_sert Tasks..."),
+	{ "InsertTasks",     "planner-stock-insert-task",    ("In_sert Tasks..."),
 	  NULL,                NULL,
 	  G_CALLBACK (gantt_view_insert_tasks_cb) },
-	{ "RemoveTask",      "planner-stock-remove-task",    N_("_Remove Task"),
-	  "<control>d",        N_("Remove the selected tasks"),
+	{ "RemoveTask",      "planner-stock-remove-task",    ("_Remove Task"),
+	  "<control>d",        ("Remove the selected tasks"),
 	  G_CALLBACK (gantt_view_remove_task_cb) },
-	{ "EditTask",        NULL,                           N_("_Edit Task Properties..."),
+	{ "EditTask",        NULL,                           ("_Edit Task Properties..."),
 	  "<shift><control>e", NULL,
 	  G_CALLBACK (gantt_view_edit_task_cb) },
-	{ "SelectAll",       NULL,                           N_("Select _All"),
-	  "<control>a",        N_("Select all tasks"),
+	{ "SelectAll",       NULL,                           ("Select _All"),
+	  "<control>a",        ("Select all tasks"),
 	  G_CALLBACK (gantt_view_select_all_cb) },
-	{ "UnlinkTask",      "planner-stock-unlink-task",    N_("_Unlink Task"),
-	  NULL,                N_("Unlink the selected tasks"),
+	{ "UnlinkTask",      "planner-stock-unlink-task",    ("_Unlink Task"),
+	  NULL,                ("Unlink the selected tasks"),
 	  G_CALLBACK (gantt_view_unlink_task_cb) },
-	{ "LinkTasks",       "planner-stock-link-task",      N_("_Link Tasks"),
-	  NULL,                N_("Link the selected tasks"),
+	{ "LinkTasks",       "planner-stock-link-task",      ("_Link Tasks"),
+	  NULL,                ("Link the selected tasks"),
 	  G_CALLBACK (gantt_view_link_tasks_cb) },
-	{ "IndentTask",      "planner-stock-indent-task",    N_("I_ndent Task"),
-	  "<shift><control>i", N_("Indent the selected tasks"),
+	{ "IndentTask",      "planner-stock-indent-task",    ("I_ndent Task"),
+	  "<shift><control>i", ("Indent the selected tasks"),
 	  G_CALLBACK (gantt_view_indent_task_cb) },
-	{ "UnindentTask",    "planner-stock-unindent-task",  N_("Unin_dent Task"),
-	  "<shift><control>u", N_("Unindent the selected tasks"),
+	{ "UnindentTask",    "planner-stock-unindent-task",  ("Unin_dent Task"),
+	  "<shift><control>u", ("Unindent the selected tasks"),
 	  G_CALLBACK (gantt_view_unindent_task_cb) },
-	{ "MoveTaskUp",      "planner-stock-move-task-up",   N_("Move Task _Up"),
-	  NULL,                N_("Move the selected tasks upwards"),
+	{ "MoveTaskUp",      "planner-stock-move-task-up",   ("Move Task _Up"),
+	  NULL,                ("Move the selected tasks upwards"),
 	  G_CALLBACK (gantt_view_move_task_up_cb) },
-	{ "MoveTaskDown",    "planner-stock-move-task-down", N_("Move Task Do_wn"),
-	  NULL,                N_("Move the selected tasks downwards"),
+	{ "MoveTaskDown",    "planner-stock-move-task-down", ("Move Task Do_wn"),
+	  NULL,                ("Move the selected tasks downwards"),
 	  G_CALLBACK (gantt_view_move_task_down_cb) },
-	{ "ResetConstraint", NULL,                           N_("Reset _Constraint"),
+	{ "ResetConstraint", NULL,                           ("Reset _Constraint"),
 	  NULL,                NULL,
 	  G_CALLBACK (gantt_view_reset_constraint_cb) },
-	{ "ZoomToFit",       GTK_STOCK_ZOOM_FIT,             N_("Zoom To _Fit"),
-	  NULL,                N_("Zoom to fit the entire project"),
+	{ "ZoomToFit",       GTK_STOCK_ZOOM_FIT,             ("Zoom To _Fit"),
+	  NULL,                ("Zoom to fit the entire project"),
 	  G_CALLBACK (gantt_view_zoom_to_fit_cb) },
-	{ "ZoomIn",          GTK_STOCK_ZOOM_IN,              N_("_Zoom In"),
-	  "<control>plus",     N_("Zoom in"),
+	{ "ZoomIn",          GTK_STOCK_ZOOM_IN,              ("_Zoom In"),
+	  "<control>plus",     ("Zoom in"),
 	  G_CALLBACK (gantt_view_zoom_in_cb) },
-	{ "ZoomOut",         GTK_STOCK_ZOOM_OUT,             N_("Zoom _Out"),
-	  "<control>minus",    N_("Zoom out"),
+	{ "ZoomOut",         GTK_STOCK_ZOOM_OUT,             ("Zoom _Out"),
+	  "<control>minus",    ("Zoom out"),
 	  G_CALLBACK (gantt_view_zoom_out_cb) },
-	{ "EditColumns",     NULL,                           N_("Edit _Visible Columns"),
-	  NULL,                N_("Edit visible columns"),
+	{ "EditColumns",     NULL,                           ("Edit _Visible Columns"),
+	  NULL,                ("Edit visible columns"),
 	  G_CALLBACK (gantt_view_edit_columns_cb) }
 };
 
 static const GtkToggleActionEntry toggle_entries[] = {
-	{ "HighlightCriticalTasks", NULL, N_("_Highlight Critical Tasks"),
+	{ "HighlightCriticalTasks", NULL, ("_Highlight Critical Tasks"),
 	  NULL, NULL,
 	  G_CALLBACK (gantt_view_highlight_critical_cb), FALSE },
-	{ "ShowGuideLines", NULL, N_("_Show Guide Lines"),
+	{ "ShowGuideLines", NULL, ("_Show Guide Lines"),
 	  NULL, NULL,
 	  G_CALLBACK (gantt_view_show_guidelines_cb), FALSE },
-	{ "NonstandardDays", NULL, N_("_Nonstandard Days"),
+	{ "NonstandardDays", NULL, ("_Nonstandard Days"),
 	  NULL, NULL,
 	  G_CALLBACK (gantt_view_nonstandard_days_cb), FALSE }
 };
diff -ur planner-0.14.6/src.orig/planner-msp-plugin.c planner-0.14.6/src/planner-msp-plugin.c
--- planner-0.14.6/src.orig/planner-msp-plugin.c	2011-09-29 15:15:45.000000000 +0100
+++ planner-0.14.6/src/planner-msp-plugin.c	2013-04-08 22:38:47.884588950 +0100
@@ -46,7 +46,7 @@
 
 
 static const GtkActionEntry entries[] = {
-	{ "MspOpen",  NULL,  N_("MS Project XML..."), NULL, N_("Import an MS Project XML file"),
+	{ "MspOpen",  NULL,  ("MS Project XML..."), NULL, ("Import an MS Project XML file"),
 	  G_CALLBACK (msp_plugin_open)
 	}
 };
diff -ur planner-0.14.6/src.orig/planner-resource-view.c planner-0.14.6/src/planner-resource-view.c
--- planner-0.14.6/src.orig/planner-resource-view.c	2011-12-25 02:54:01.000000000 +0000
+++ planner-0.14.6/src/planner-resource-view.c	2013-04-09 08:12:59.291702469 +0100
@@ -230,29 +230,29 @@
 static PlannerViewClass *parent_class = NULL;
 
 static const GtkActionEntry entries[] = {
-	{ "InsertResource",   "planner-stock-insert-resource", N_("_Insert Resource"),
-	  "<Control>i",        N_("Insert a new resource"),
+	{ "InsertResource",   "planner-stock-insert-resource", ("_Insert Resource"),
+	  "<Control>i",        ("Insert a new resource"),
 	  G_CALLBACK (resource_view_insert_resource_cb) },
-	{ "InsertResources",  "planner-stock-insert-resource", N_("In_sert Resources..."),
+	{ "InsertResources",  "planner-stock-insert-resource", ("In_sert Resources..."),
 	  NULL,                NULL,
 	  G_CALLBACK (resource_view_insert_resources_cb) },
-	{ "RemoveResource",   "planner-stock-remove-resource", N_("_Remove Resource"),
-	  "<Control>d",        N_("Remove the selected resource"),
+	{ "RemoveResource",   "planner-stock-remove-resource", ("_Remove Resource"),
+	  "<Control>d",        ("Remove the selected resource"),
 	  G_CALLBACK (resource_view_remove_resource_cb) },
-	{ "EditResource",     GTK_STOCK_PROPERTIES,            N_("_Edit Resource Properties..."),
+	{ "EditResource",     GTK_STOCK_PROPERTIES,            ("_Edit Resource Properties..."),
 	  "<Shift><Control>e", NULL,
 	  G_CALLBACK (resource_view_edit_resource_cb) },
-	{ "EditGroups",       "planner-stock-edit-groups",     N_("Edit _Groups"),
-	  NULL,                N_("Edit resource groups"),
+	{ "EditGroups",       "planner-stock-edit-groups",     ("Edit _Groups"),
+	  NULL,                ("Edit resource groups"),
 	  G_CALLBACK (resource_view_edit_groups_cb) },
-	{ "SelectAll",        NULL,                            N_("Select _All"),
-	  "<Control>a",        N_("Select all tasks"),
+	{ "SelectAll",        NULL,                            ("Select _All"),
+	  "<Control>a",        ("Select all tasks"),
 	  G_CALLBACK (resource_view_select_all_cb) },
-	{ "EditCustomProps",  GTK_STOCK_PROPERTIES,            N_("Edit _Custom Properties..."),
+	{ "EditCustomProps",  GTK_STOCK_PROPERTIES,            ("Edit _Custom Properties..."),
 	  NULL,                NULL,
 	  G_CALLBACK (resource_view_edit_custom_props_cb) },
-	{ "EditColumns",     NULL,                           N_("Edit _Visible Columns"),
-	  NULL,                N_("Edit visible columns"),
+	{ "EditColumns",     NULL,                           ("Edit _Visible Columns"),
+	  NULL,                ("Edit visible columns"),
 	  G_CALLBACK (resource_view_edit_columns_cb) }
 };
 
@@ -266,13 +266,13 @@
 #define GIF_CB(x) ((GtkItemFactoryCallback)(x))
 
 static GtkItemFactoryEntry popup_menu_items[] = {
-	{ N_("/_Insert resource"),   NULL, GIF_CB (resource_view_popup_insert_resource_cb),
+	{ ("/_Insert resource"),   NULL, GIF_CB (resource_view_popup_insert_resource_cb),
 	  POPUP_INSERT,  "<Item>",       NULL
 	},
-	{ N_("/_Remove resource"),   NULL, GIF_CB (resource_view_popup_remove_resource_cb),
+	{ ("/_Remove resource"),   NULL, GIF_CB (resource_view_popup_remove_resource_cb),
 	  POPUP_REMOVE,  "<StockItem>",  GTK_STOCK_DELETE
 	},
-	{ N_("/_Edit resource..."),  NULL, GIF_CB (resource_view_popup_edit_resource_cb),
+	{ ("/_Edit resource..."),  NULL, GIF_CB (resource_view_popup_edit_resource_cb),
 	  POPUP_EDIT,    "<Item>",       NULL }
 };
 
diff -ur planner-0.14.6/src.orig/planner-sql-plugin.c planner-0.14.6/src/planner-sql-plugin.c
--- planner-0.14.6/src.orig/planner-sql-plugin.c	2011-12-22 22:53:44.000000000 +0000
+++ planner-0.14.6/src/planner-sql-plugin.c	2013-04-08 22:39:15.970587309 +0100
@@ -91,11 +91,11 @@
 };
 
 static const GtkActionEntry entries[] = {
-	{ "Open",  NULL,  N_("Open from Database..."),
-	  NULL, N_("Open a project from a database"),
+	{ "Open",  NULL,  ("Open from Database..."),
+	  NULL, ("Open a project from a database"),
 	  G_CALLBACK (sql_plugin_open) },
-	{ "Save",  NULL,  N_("Save to Database"),
-	  NULL, N_("Save the current project to a database"),
+	{ "Save",  NULL,  ("Save to Database"),
+	  NULL, ("Save the current project to a database"),
 	  G_CALLBACK (sql_plugin_save) }
 };
 
diff -ur planner-0.14.6/src.orig/planner-task-view.c planner-0.14.6/src/planner-task-view.c
--- planner-0.14.6/src.orig/planner-task-view.c	2011-09-29 15:15:45.000000000 +0100
+++ planner-0.14.6/src/planner-task-view.c	2013-04-09 08:12:42.422703454 +0100
@@ -116,54 +116,54 @@
 static PlannerViewClass *parent_class = NULL;
 
 static const GtkActionEntry entries[] = {
-	{ "InsertTask",      "planner-stock-insert-task",      N_("_Insert Task"),
-	  "<Control>i",        N_("Insert a new task"),
+	{ "InsertTask",      "planner-stock-insert-task",      ("_Insert Task"),
+	  "<Control>i",        ("Insert a new task"),
           G_CALLBACK (task_view_insert_task_cb) },
-	{ "InsertTasks",     "planner-stock-insert-task",      N_("In_sert Tasks..."),
+	{ "InsertTasks",     "planner-stock-insert-task",      ("In_sert Tasks..."),
 	  NULL,                NULL,
 	  G_CALLBACK (task_view_insert_tasks_cb) },
-	{ "RemoveTask",      "planner-stock-remove-task",      N_("_Remove Task"),
-	  "<Control>d",        N_("Remove the selected tasks"),
+	{ "RemoveTask",      "planner-stock-remove-task",      ("_Remove Task"),
+	  "<Control>d",        ("Remove the selected tasks"),
 	  G_CALLBACK (task_view_remove_task_cb) },
-	{ "EditTask",        NULL,                             N_("_Edit Task"),
+	{ "EditTask",        NULL,                             ("_Edit Task"),
 	  "<Shift><Control>e", NULL,
 	  G_CALLBACK (task_view_edit_task_cb) },
-	{ "SelectAll",       NULL,                             N_("Select _All"),
-	  "<Control>a",        N_("Select all tasks"),
+	{ "SelectAll",       NULL,                             ("Select _All"),
+	  "<Control>a",        ("Select all tasks"),
 	  G_CALLBACK (task_view_select_all_cb) },
-	{ "UnlinkTask",      "planner-stock-unlink-task",      N_("_Unlink Task"),
-	  NULL,                N_("Unlink the selected tasks"),
+	{ "UnlinkTask",      "planner-stock-unlink-task",      ("_Unlink Task"),
+	  NULL,                ("Unlink the selected tasks"),
 	  G_CALLBACK (task_view_unlink_task_cb) },
-	{ "LinkTasks",       "planner-stock-link-task",        N_("_Link Tasks"),
-	  NULL,                N_("Link the selected tasks"),
+	{ "LinkTasks",       "planner-stock-link-task",        ("_Link Tasks"),
+	  NULL,                ("Link the selected tasks"),
 	  G_CALLBACK (task_view_link_tasks_cb) },
-	{ "IndentTask",      "planner-stock-indent-task",      N_("I_ndent Task"),
-	  "<Shift><Control>i", N_("Indent the selected tasks"),
+	{ "IndentTask",      "planner-stock-indent-task",      ("I_ndent Task"),
+	  "<Shift><Control>i", ("Indent the selected tasks"),
 	  G_CALLBACK (task_view_indent_task_cb) },
-	{ "UnindentTask",    "planner-stock-unindent-task",    N_("Unin_dent Task"),
-	  "<Shift><Control>u", N_("Unindent the selected tasks"),
+	{ "UnindentTask",    "planner-stock-unindent-task",    ("Unin_dent Task"),
+	  "<Shift><Control>u", ("Unindent the selected tasks"),
 	  G_CALLBACK (task_view_unindent_task_cb) },
-	{ "MoveTaskUp",      "planner-stock-move-task-up",     N_("Move Task _Up"),
-	  NULL,                N_("Move the selected tasks upwards"),
+	{ "MoveTaskUp",      "planner-stock-move-task-up",     ("Move Task _Up"),
+	  NULL,                ("Move the selected tasks upwards"),
 	  G_CALLBACK (task_view_move_task_up_cb) },
-	{ "MoveTaskDown",    "planner-stock-move-task-down",   N_("Move Task Do_wn"),
-	  NULL,                N_("Move the selected tasks downwards"),
+	{ "MoveTaskDown",    "planner-stock-move-task-down",   ("Move Task Do_wn"),
+	  NULL,                ("Move the selected tasks downwards"),
 	  G_CALLBACK (task_view_move_task_down_cb) },
-	{ "ResetConstraint", "planner-stock-reset-constraint", N_("Reset _Constraint"),
+	{ "ResetConstraint", "planner-stock-reset-constraint", ("Reset _Constraint"),
           NULL,                NULL,
 	  G_CALLBACK (task_view_reset_constraint_cb) },
-	{ "EditCustomProps", GTK_STOCK_PROPERTIES,             N_("_Edit Custom Properties..."),
+	{ "EditCustomProps", GTK_STOCK_PROPERTIES,             ("_Edit Custom Properties..."),
 	  NULL,                NULL,
 	  G_CALLBACK (task_view_edit_custom_props_cb) },
-	{ "EditColumns",       NULL,                           N_("Edit _Visible Columns"),
-	  NULL,                N_("Edit visible columns"),
+	{ "EditColumns",       NULL,                           ("Edit _Visible Columns"),
+	  NULL,                ("Edit visible columns"),
 	  G_CALLBACK (task_view_edit_columns_cb) }
 };
 
 static const GtkToggleActionEntry toggle_entries[] = {
-	{ "HighlightCriticalTasks", NULL, N_("_Highlight Critical Tasks"), NULL, NULL,
+	{ "HighlightCriticalTasks", NULL, ("_Highlight Critical Tasks"), NULL, NULL,
 	  G_CALLBACK (task_view_highlight_critical_cb), FALSE },
-	{ "NonstandardDays", NULL, N_("_Nonstandard Days"), NULL, NULL,
+	{ "NonstandardDays", NULL, ("_Nonstandard Days"), NULL, NULL,
 	  G_CALLBACK (task_view_nonstandard_days_cb), FALSE }
 };
 
diff -ur planner-0.14.6/src.orig/planner-usage-view.c planner-0.14.6/src/planner-usage-view.c
--- planner-0.14.6/src.orig/planner-usage-view.c	2011-12-25 02:54:01.000000000 +0000
+++ planner-0.14.6/src/planner-usage-view.c	2013-04-09 08:12:18.015704880 +0100
@@ -105,17 +105,17 @@
 static PlannerViewClass *parent_class = NULL;
 
 static const GtkActionEntry entries[] = {
-        { "ZoomOut",   GTK_STOCK_ZOOM_OUT, N_("Zoom out"),
-	  NULL, N_("Zoom out"),
+        { "ZoomOut",   GTK_STOCK_ZOOM_OUT, ("Zoom out"),
+	  NULL, ("Zoom out"),
 	  G_CALLBACK (usage_view_zoom_out_cb) },
-        { "ZoomIn",    GTK_STOCK_ZOOM_IN,  N_("Zoom in"),
-	  NULL, N_("Zoom in"),
+        { "ZoomIn",    GTK_STOCK_ZOOM_IN,  ("Zoom in"),
+	  NULL, ("Zoom in"),
 	  G_CALLBACK (usage_view_zoom_in_cb) },
-        { "ZoomToFit", GTK_STOCK_ZOOM_FIT, N_("Zoom to fit"),
-	  NULL, N_("Zoom to fit the entire project"),
+        { "ZoomToFit", GTK_STOCK_ZOOM_FIT, ("Zoom to fit"),
+	  NULL, ("Zoom to fit the entire project"),
 	  G_CALLBACK (usage_view_zoom_to_fit_cb) },
-	{ "EditColumns", NULL, N_("Edit _Visible Columns"),
-	  NULL, N_("Edit visible columns"),
+	{ "EditColumns", NULL, ("Edit _Visible Columns"),
+	  NULL, ("Edit visible columns"),
 	  G_CALLBACK (usage_view_edit_columns_cb) }
 };
 
--- planner-0.14.6/src.orig/planner-xml-planner-plugin.c	2013-04-08 22:20:17.869653806 +0100
+++ planner-0.14.6/src/planner-xml-planner-plugin.c	2013-04-08 22:20:43.927652283 +0100
@@ -51,8 +51,8 @@
 
 static const GtkActionEntry entries[] = {
 	{ "XML Planner Export", NULL,
-	  N_("Planner 0.11 Format"), NULL,
-	  N_("Export project to a file suitable for Planner 0.11"),
+	  "Planner 0.11 Format", NULL,
+	  "Export project to a file suitable for Planner 0.11",
 	  G_CALLBACK (xml_planner_plugin_export) }
 };
 
--- planner-0.14.6/src.orig/planner-html-plugin.c	2013-04-08 21:45:12.663588454 +0100
+++ planner-0.14.6/src/planner-html-plugin.c	2013-04-08 22:11:33.859684423 +0100
@@ -43,8 +43,8 @@
 
 static const GtkActionEntry action_entries[] = {
 	{ "HTML Export", NULL,
-	  N_("HTML"), NULL,
-	  N_("Export project to HTML"),
+	  "HTML", NULL,
+	  "Export project to HTML",
 	  G_CALLBACK (html_plugin_export)
 	}
 };
@@ -76,7 +76,7 @@
 						 GTK_DIALOG_DESTROY_WITH_PARENT,
 						 GTK_MESSAGE_ERROR,
 						 GTK_BUTTONS_OK,
-						 _("Could not export to HTML"));
+						 "%s", _("Could not export to HTML"));
 		gtk_dialog_run (GTK_DIALOG (dialog));
 		gtk_widget_destroy (dialog);
 	}
--- planner-0.14.6/src.orig/planner-application.c	2011-12-22 22:53:44.000000000 +0000
+++ planner-0.14.6/src/planner-application.c	2013-04-09 08:32:14.312634984 +0100
@@ -51,77 +51,77 @@
 static GtkStockItem stock_items[] = {
 	{
 		"planner-stock-insert-task",
-		N_("Insert"),
+		("Insert"),
 		0, /*GdkModifierType*/
 		0, /*keyval*/
 		GETTEXT_PACKAGE
 	},
 	{
 		"planner-stock-remove-task",
-		N_("Remove"),
+		("Remove"),
 		0, /*GdkModifierType*/
 		0, /*keyval*/
 		GETTEXT_PACKAGE
 	},
 	{
 		"planner-stock-unlink-task",
-		N_("Unlink"),
+		("Unlink"),
 		0, /*GdkModifierType*/
 		0, /*keyval*/
 		GETTEXT_PACKAGE
 	},
 	{
 		"planner-stock-link-tasks",
-		N_("Link"),
+		("Link"),
 		0, /*GdkModifierType*/
 		0, /*keyval*/
 		GETTEXT_PACKAGE
 	},
 	{
 		"planner-stock-indent-task",
-		N_("Indent"),
+		("Indent"),
 		0, /*GdkModifierType*/
 		0, /*keyval*/
 		GETTEXT_PACKAGE
 	},
 	{
 		"planner-stock-unindent-task",
-		N_("Unindent"),
+		("Unindent"),
 		0, /*GdkModifierType*/
 		0, /*keyval*/
 		GETTEXT_PACKAGE
 	},
 	{
 		"planner-stock-move-task-up",
-		N_("Move up"),
+		("Move up"),
 		0, /*GdkModifierType*/
 		0, /*keyval*/
 		GETTEXT_PACKAGE
 	},
 	{
 		"planner-stock-move-task-down",
-		N_("Move down"),
+		("Move down"),
 		0, /*GdkModifierType*/
 		0, /*keyval*/
 		GETTEXT_PACKAGE
 	},
 	{
 		"planner-stock-insert-resource",
-		N_("Insert"),
+		("Insert"),
 		0, /*GdkModifierType*/
 		0, /*keyval*/
 		GETTEXT_PACKAGE
 	},
        	{
 		"planner-stock-remove-resource",
-		N_("Remove"),
+		("Remove"),
 		0, /*GdkModifierType*/
 		0, /*keyval*/
 		GETTEXT_PACKAGE
 	},
 	{
 		"planner-stock-edit-groups",
-		N_("Edit"),
+		("Edit"),
 		0, /*GdkModifierType*/
 		0, /*keyval*/
 		GETTEXT_PACKAGE
--- planner-0.14.6/src.orig/planner-main.c	2011-10-14 16:38:41.000000000 +0100
+++ planner-0.14.6/src/planner-main.c	2013-04-09 08:54:18.599557609 +0100
@@ -36,8 +36,8 @@
 static gchar **args_remaining = NULL;
 
 static GOptionEntry options[] = {
-		{ "geometry", 'g', 0, G_OPTION_ARG_STRING, &geometry, N_("Create the initial window with the given geometry."), N_("GEOMETRY")},
-		{ G_OPTION_REMAINING, 0, 0, G_OPTION_ARG_FILENAME_ARRAY, &args_remaining, NULL, N_("FILES") },
+		{ "geometry", 'g', 0, G_OPTION_ARG_STRING, &geometry, ("Create the initial window with the given geometry."), ("GEOMETRY")},
+		{ G_OPTION_REMAINING, 0, 0, G_OPTION_ARG_FILENAME_ARRAY, &args_remaining, NULL, ("FILES") },
 		{ NULL }
 	};
 
